scarlet/blend.py

```diff
@@ -21,18 +22,35 @@ class ScarletRestartException(Exception):
     """
     pass
 
-class Blend(object):
-    """The blended scene as interpreted by the deblender.
+class Blend(ComponentTree):
+    """The blended scene.
+
+    The class represents a celestial scene and provides the functions to fit it
+    to data.
     """
-    def __init__(self, sources, img, weights=None, bg_rms=None, config=None):
+    def __init__(self, components):
         """Constructor
 
+        Form a blended scene from a collection of `~scarlet.component.Component`s
+
+        Parameters
+        ----------
+        components: list of `~scarlet.component.Component` or `~scarlet.component.ComponentTree`
+        """
+
+        super(Blend, self).__init__(components)
+
+        # check bands and PSFs
+        self.B = self.components[0].B
```

Comment:
It might be good to implement a check that all of the components have the same `B` value